rm-datasource/src/main/java/io/seata/rm/datasource/undo/KeywordCheckerFactory.java

```diff
@@ -39,8 +39,9 @@ public static KeywordChecker getKeywordChecker(String dbType) {
         if (KEYWORD_CHECKER_MAP.get(dbType) != null) {
             return KEYWORD_CHECKER_MAP.get(dbType);
         }
-        KeywordChecker tableMetaCache = EnhancedServiceLoader.load(KeywordChecker.class, dbType);
+        KeywordChecker tableMetaCache =
+                EnhancedServiceLoader.load(KeywordChecker.class, dbType);
```

Comment:
I think there is no uniform format style yet, so it's better to keep it unchanged.

Reply:
Restored